wom/src/test/scala/wom/util/YamlUtilsSpec.scala

```diff
@@ -74,12 +74,6 @@ class YamlUtilsSpec extends FlatSpec with Matchers with TableDrivenPropertyCheck
       refineMV[NonNegative](0),
       null,
     ),
-    (
-      "an empty yaml",
-      "",
-      refineMV[NonNegative](1),
-      "null",
-    ),
```

Comment:
A cursory google search tells me that an empty file is valid yaml(1)(2).  Not totally sure why this test was created in the first place?

1: https://stackoverflow.com/questions/28628511/read-empty-yaml-file
2: https://yaml.org/spec/1.0/#id2561559

Reply:
FYI- the test was created as a wrapper was being added aroud circe-yaml. The "legal" and "illegal" refer to the underlying implementation allowing the yaml, not the legality as defined by the spec. The wrapper was added because circe-yaml was throwing a StackOverflowError on bad yaml, killing the JVM instead of throwing an exception.

Now that circe-yaml is not returning an error, the yaml is now "legal"